PR_REVIEW_CHECKLIST.md

```diff
@@ -0,0 +1,98 @@
+# PR review checklist
+
+This document lists specific checks that should be done when reviewing a PR. This means these items should be checked by:
+1. Every contributor submitting a Pull Request
+2. Every C+ (Contributor Plus) team member reviewing a PR
+3. Every Expensify employee submitting / reviewing a PR
+
+This is a living document, so we will be updating this from time to time. Feel free to suggest additions or changes in the #expensify-open-source Slack channel.
+
+# Check PR template
+
+1. Make sure the PR template includes screenshots or videos of the changes being tested on **all platforms**. If there are one or more platforms missing, follow up with the contributor to determine why - we shouldn't be merging changes that aren't tested everywhere.
+    - Feel free to ask for help on how to test changes on all platforms.
+2. Make sure the “Fixed Issues” section is formatted correctly. Links should be clickable and lead to the correct issue being fixed by this PR.
+3. Make sure the testing steps are clear and correctly test the changes made. Make note of what’s required for the test account (i.e. do you need to test using a brand new account, have a workspace, have a bank account, etc).
+    - > **Rule of thumb**: Over-explain what the tester needs to do. Do not rely on the tester having existing knowledge of the app.
+      > 
+      > ### Example of bad steps:
+      >
+	  >	1. Go to this page
+	  >	2. Click the button
+      >
+      > ### Example of good steps:
+      >
+      > 1. Log in with an account that has a workspace, or follow `[these steps](link-to-instructions-for setting-up-workspace)` to make one.
+      > 2. Click the settings button on the top left (the one with your avatar)
+      > 3. Navigate to “Your Workspace” > Reimburse Receipts > Connect bank account
+      > 4. Verify that the bank account view opens without errors
+      >
+    - Take note of the distinction between testing _locally_ and _on staging_. Note: The staging site references the production version of the API.
+4. Make sure the testing steps also cover possible regressions.
+    1. Cases where we definitely need new regression tests:
+        1. Building a new page
+        2. Building a new feature
+        3. Building a new component?
+    2. Cases where we might need new regression tests:
+        1. Bug was easy to find
```

Comment:
I still think this section is confusing. How does a contributor know they aren't suggesting regressions we already have? I think this should be between internal engineers and Applause for now. Just my 2 cents though.

Reply:
Should we at least move it into the next section? We are talking about testing here and also in the `Testing the changes` section. Maybe we can find a better way to organize this information?

Reply:
Regression discussion on hold, asking in #applause-internal if Applause may be able to take of this for us & for contributors

Reply:
Removing for now, regression testing discussion pushed to future haha